.github/actions/transition/action.yml

```diff
@@ -8,7 +8,9 @@ runs:
     - run: npx create-typescript-app
       shell: bash
     - id: auto-commit-action
-      uses: stefanzweifel/git-auto-commit-action@e348103e9026cc0eee72ae06630dbe30c8bf7a79 # v5
+      uses: >-
+        stefanzweifel/git-auto-commit-action@e348103e9026cc0eee72ae06630dbe30c8bf7a79
+        # v5
```

Comment:
🐛 The YAML printer should not be line-breaking before it detects and fixes for comments...

Reply:
https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2097